config/owasp/suppressions.xml

```diff
@@ -708,4 +708,11 @@ file name: ant*.jar: readline.jar
         <cve>CVE-2020-5407</cve>
         <cve>CVE-2020-5408</cve>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: postgresql-42.2.8.jar
+   ]]></notes>
```

Comment:
can't we avoid this with bump up postgress?

Reply:
Can upgrade this version, PR fails in flyway scripts